src/Microsoft.DotNet.Build.CloudTestTasks/RunnerScripts/scriptrunner/scriptrunner.py

```diff
@@ -81,8 +83,9 @@ def _main(settings, optlist, args):
         else:
             log.error("Error: No exception thrown, but XUnit results not created")
             test_executor.report_error(settings, failure_type="XUnitTestFailure")
+            return WORKITEM_FAILED
```

Comment:
Currently it doesn't but once we update to newer XUnit sources the values will be either 0-5, and anything outside of that is a crash.